docs/search/getting-started.md

```diff
@@ -0,0 +1,68 @@
+---
+sidebar_position: 20
+title: Getting started with Search
+---
+
+In this getting started guide we will be adding search functionality to the Address Book app that you build by following our [quickstart guide](https://macrometa.com/docs/quickstart).
+
+## Prerequisites
+
+- A [Macrometa account](https://auth.paas.macrometa.io/).
+- Completion of our [quickstart guide](https://macrometa.com/docs/quickstart).
+
+## Step 1: Create a view
+
+ A search _view_ an inverted index that contains the search configuration and all document attributes across one or many collections.
+
+1. On the side menu, click **Search Views**.
+2. Click **New view**.
+3. In **Name**, enter `addresses_view`.
+4. Click the **plus** under Mapping Definition.
+5. Select the `addresses` collection from the COLLECTION dropdown. 
+6. Enter `lastName` in `FIELD` and select **text_en** in the ANALYZER dropdown. 
+7. Follow steps 5 and 6 two more times, but add `firstName` in the second Mapping Definition and `email` in the third Mapping Definition.
```

Comment:
You do not need to capitalize things unless you are referring to the UI. In step 4, you were referring to the UI. In this step, they are generic mapping definitions, so do not require capitalization.